gpu-aware-scheduling/deploy/gas-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gpu-aware-scheduling
+  namespace: default 
+  labels:
+    app: gas
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gas
+  template:
+    metadata:
+      labels:
+        app: gas
+    spec:
+      serviceAccountName: gpu-aware-scheduling-service-account
+      containers:
+      - name: gasext
+        command:
+        - /extender
+        - --cert=/gas/cert/tls.crt
+        - --key=/gas/cert/tls.key
+        - --cacert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        - -v
+        - "4"
```

Comment:
--v=4 ?

Reply:
yes, although flags can be single dash/minus, there's no real reason to separate them. I had some issues with the args and that is a leftover, I'm sure I can clean that up